src/components/Settings/integrations/Google/Modals/EditGoogleAccountModal.test.tsx

```diff
@@ -332,11 +332,13 @@ describe('EditGoogleAccountModal', () => {
     );
 
     await waitFor(() =>
-      expect(getByTestId('Call-Checkbox')).toBeInTheDocument(),
+      expect(
+        getByTestId('APPOINTMENT_VIDEO_CALL-Checkbox'),
+      ).toBeInTheDocument(),
     );
 
     await act(async () => {
```

Comment:
Let's not wrap this block of code in an `act` call. `userEvent.xxx` methods already use `act` internally, and `waitFor`s shouldn't be inside `act`.